pkg/controller/clusterdeprovision/clusterdeprovision_controller.go

```diff
@@ -213,6 +228,52 @@ func (r *ReconcileClusterDeprovision) Reconcile(request reconcile.Request) (reco
 		return reconcile.Result{}, nil
 	}
 
+	actuator := r.getActuator(instance)
+	if actuator == nil {
+		rLog.Debug("No actuator found for this provider")
+	} else {
+		// actuator found, ensure creds work.
+		err := actuator.TestCredentials(instance, r.Client, rLog)
+		if err != nil {
+			rLog.WithError(err).Error("Credential check failed")
```

Comment:
This should be a warning or an info, not an error. This is a problem with user input not with the functioning of Hive.